soc/arm/nordic_nrf/nrf91/dts_fixup.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* SoC level DTS fixup file */
+
+#define DT_NUM_IRQ_PRIO_BITS        DT_ARM_V8M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define DT_ADC_0_NAME                   DT_NORDIC_NRF_SAADC_ADC_0_LABE
```

Comment:
One character (`L`) at the end of the line got eaten somehow.

Reply:
Fixing it, thanks.